frontend/src/Frontend/UI/DeploymentSettings.hs

```diff
@@ -961,7 +961,7 @@ uiSenderCapabilities m cid mCaps mkSender = do
           rest <- staticCapabilityRows (Just <$> eApplyToAll) $ filter (not . isGas . _dappCap_cap) caps
           pure ( _capabilityInputRow_account gas
                , combineMaps [(_capabilityInputRow_value gas),rest]
-               , constDyn 0
+               , constDyn (1 {- Gas payer -} + length caps)
                )
```

Comment:
Would the counting be clearer/more robust if we did something like
```haskell
(rest, restCount) <- case mCampf of 
  (...)
pure ( _capabilityInputRow_account gas
     , combineMaps [(_capabilityInputRow_value gas), rest]
     , liftA2 (+) (1 <$ rest) (maybe 0 length <$> mcaps)
     )
```
?

Reply:
`mCaps` isn't a `Dynamic`, otherwise yeah that would be better.